commands/util/pvp-rank.js

```diff
@@ -0,0 +1,161 @@
+const log = require('loglevel').getLogger('Counters'),
+  Commando = require('discord.js-commando'),
+  {MessageEmbed} = require('discord.js'),
+  {CommandGroup} = require('../../app/constants'),
+  fetch = require('node-fetch'),
+  cheerio = require('cheerio'),
+  Utility = require('../../app/utility');
+
+class PvPRankingData {
+  constructor(pokemon, league, atk, def, sta, filter){
+    this.inputPokemon = pokemon;
+    this.inputLeague = league;
+    this.inputATKiv = atk;
+    this.inputDEFiv = def;
+    this.inputSTAiv = sta;
+    if(typeof(filter) == 'undefined'){
+      this.inputFilter = "0";
+    }
+    else{
+      this.inputFilter = filter;
+    }
+  }
+
+  async scrape(){
+    String.prototype.titleCase = function () {
+      return this.replace(/\w\S*/g, function(str) {return str.charAt(0).toUpperCase() + str.substr(1).toLowerCase();});
+    };
+    this.cpLeague;
+    let ivFilter;
+    if(this.inputLeague == 'ultra'){

```

Comment:
Since you're defaulting this to `1500` no matter what, this can be simplified to:

`this.cpLeague = this.inputLeague == 'ultra' ? "2500" : "1500";`

Reply:
I'll test this out- I'm unfamiliar with this. Thanks!